velox/docs/monthly-updates/december-2021.rst

```diff
@@ -0,0 +1,43 @@
+********************
+December 2021 Update
+********************
+
+Documentation
+-------------
+
+* Add :doc:`Aggregation-related optimizations <../develop/aggregations>` guide.
+* Add missing documentation for :func:`json_extract_scalar` function.
+* Add documentation for missing functions in :doc:`Mathematical Functions <../functions/math>` and :doc:`Comparison Functions <../functions/comparison>`.
+* Update :doc:`How to add an aggregate function? <../develop/aggregate-functions>` guide to reflect changes to the function registry.
+* Update :doc:`How to add a scalar function? <../develop/scalar-functions>` guide to describe how to write functions with complex type inputs and results.
+
+
+Core Library
+------------
+
+* Add support for partition key aliases to Hive Connector.
+* Add support for conversions between Velox string and Arrow string.
+* Add :ref:`StreamingAggregation <AggregationNode and StreamingAggregationNode>` operator.
+* Fix a bug in HashJoin filter to enable TPC-H query 19.
+
+
+Presto Functions
+----------------
+
+* Add :func:`corr`, :func:`covar_samp`, :func:`covar_pop`, :func:`every` aggregate functions.
+* Add :func:`from_base`, :func:`array_position` functions.
+* Add support for 'timestamp with timezone' in :func:`to_unixtime` function.
+
+
+Spark Functions
+----------------
+* Add :func:`sort_array`, :func:`array_sort` functions.
```

Comment:
I don't think these links work as we don't have documentation for Spark functions. CC: @funrollloops 

To unblock this PR, perhaps, remove the Spark Functions section.

Reply:
Done. I should have checked the links. Thanks for all the feedback!